code/WorkInProgress/Mini/ATM.dm

```diff
@@ -362,6 +363,52 @@ log transactions
 					playsound(loc, 'sound/items/polaroid1.ogg', 50, 1)
 				else
 					playsound(loc, 'sound/items/polaroid2.ogg', 50, 1)
+			if("vending_machine")
+				if(authenticated_account)
+					if(world.timeofday < lastprint + PRINT_DELAY)
+						usr << "<span class='notice'>The [src.name] flashes an error on its display.</span>"
```

Comment:
@ComicIronic
Yeah, that was a problem I'd thought about too. It was in the original ATM code. I'll make it more clear.
@Iamgoofball
I'm fairly new to byond code; why is it unsafe and how can I make it safe?